### PR TITLE
Adding Admin Page

### PR DESCRIPTION
**Pull request by lloza1998**
The overall changes include

- Creating the basic layout of adminpage.jsp
- Creating a servlet to deal with the new page
- Adding link to the navigation bar
- Adding tests for new page

### CODE REVIEW

**Review by cariygan — APPROVED** [review]
nice

**Review by Linda-Yang — APPROVED** [review]
:)

**Review by jbhesania — APPROVED** [review]
Good!

File: src/main/java/codeu/controller/AdminPageServlet.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package codeu.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet class responsible for user registration.
```
Comment (jbhesania) [inline src/main/java/codeu/controller/AdminPageServlet.java:10]: Just a quick note to remember to change the comments here!

**Pull request merged**